app/models/concerns/scorable.rb

```diff
@@ -0,0 +1,13 @@
+module Scorable
+  extend ActiveSupport::Concern
+
+  def scored(recent_boundary: 1.month.ago)
```

Comment:
This needs to be the scores for the current academic year, not the scores from the last month.

Reply:
:+1: